src/components/Button/Button.tsx

```diff
@@ -106,7 +107,10 @@ const button = ({
     : null;
 
   const iconMarkup = icon
-    ? <span className={theme.customIcon}><Icon source={icon} /></span>
+    ? <span className={theme.customIcon}>
+        <Icon source={icon} />
+        {accessibilityLabel && <VisuallyHidden>{accessibilityLabel}</VisuallyHidden> || title && <VisuallyHidden>{title}</VisuallyHidden>}
```

Comment:
@Farhan-emgage this needs a logical check to ensure either `accessibilityLabel` or `title` prop exists. It also needs a logical check to only include the `title` as VisuallyHidden in the case that both exist. Please have an FE team member assist you if needed.

Reply:
@weegrom 

above changes are done